x-pack/plugins/cases/server/client/utils.test.ts

```diff
@@ -166,20 +61,11 @@ describe('utils', () => {
           "closed_at": null,
           "closed_by": null,
           "connector": Object {
-            "fields": Array [
-              Object {
-                "key": "issueType",
-                "value": "Task",
-              },
-              Object {
-                "key": "priority",
-                "value": "High",
-              },
-              Object {
-                "key": "parent",
-                "value": null,
-              },
-            ],
+            "fields": Object {
```

Comment:
`transformNewCase` leaves the connector as is because the service layer will transform it once passed.